atom/common/node_bindings.cc

```diff
@@ -194,6 +199,10 @@ node::Environment* NodeBindings::CreateEnvironment(
       base::CommandLine::ForCurrentProcess()->HasSwitch("debug-brk"))
     process.Set("_debugWaitConnect", true);
 
+  process.SetMethod(
+    "setProcessTitle",
```

Comment:
Since `process.title` works on Linux, and there is no need for Windows, I think we only need to provide a Mac only API.